data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3839 @@
+				accept
+	to fail
+		has "Unfettered: Close Quarter: done"
+	on complete
+		conversation
+			`You cannot find Admiral Danforth, and wager he's probably busy with the pirates above. You think of disturbing him, but another Oathbreaker sees you and asks what you're looking for. You start speaking with him of the Pug war, and recognizing you, he seems eager to share his experience.`
+			`	"I was a simple soldier fighting with the rest of the fleet, in the Republic, at that time. We managed to find an isolated and disabled Pug ship. That by itself seemed like an opportunity too good to let go. Given we were in a Cruiser, we did not fear the crew of such a small ship. But we were wrong: as soon as we boarded said ship, we knew something was wrong. But we were committed, so we went in anyway. Being a soldier, I was on the frontline, and I saw things... Things you do not want to see, trust me. I saw people falling for no apparent reason. But that was not the worst of it. As you can see, I have survived, and that by itself is terrifying. They shot me! They shot me, and yet I lived."`
+			`	You thank him for telling you this story and then think to yourself. This means there is a good chance the Pug have the non-lethal weapons you are looking for. Now you just have to find the Pug, but they'll probably not give you these weapons willingly...`
+
+
+
+mission "Unfettered: Know Your Enemy"
+	name `Acquire a translator`
+	description `You should use your pseudo-independence from the Unfettered to scan Wanderer space. Also, you need to obtain a translator. First go meet the Wanderers, and then go to <stopovers> to see if He Yi managed to find one.`
+	source "Firelode"
+	stopover "Warfeed"
+	to offer
+		has "Unfettered: Solifuge 9: Shield Beetles Mass Production: done"
+	on offer
+		conversation
+			`Moving through the spaceport, you notice some buildings are not as dilapidated as before. You wonder if the Unfettered have found new interest in restoring their past, or if the new Hai arrivals brought this about. Whatever the case is, this does not look like a place of conflict anymore. It looks like a place of hope.`
+			`	An Unfettered herald approaches and notices you looking around. "Incredible, is it not? To see hope on the face of buildings and on the face of people. I am here to say that it's time you learn how to communicate with us - with the True language of the Hai. Not only with us. You also need to be able to make demands to the birds, and there is only one appropriate language for it. We have learned that they too speak Hai, even if they disgrace our language by doing so."`
+			branch speakhai
+				not "language: Wanderer"
+			choice
+				`	"I already speak Hai. I have a translator, but I did't show it to you yet because I didn't know how you would react."`
```

Comment:
If the `<first>` is claiming they chose to hide the translator, that should be an option previously presented to the player.

Reply:
That could be complicated but you're right

Reply:
Actually he wouldnt know if you used it, as its currently written. So no need to say you've hidden it, they just haven't seen you use it yet because you didnt need it.

Reply:
I changed it. If you have the translator you won't get this mission.